extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -0,0 +1,467 @@
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as nls from 'vscode-nls';
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as os from 'os';
+import { SchemaCompareResult } from '../schemaCompareResult';
+
+const localize = nls.loadMessageBundle();
+
+export class SchemaCompareDialog {
+
+	private static readonly CompareButtonText: string = localize('schemaCompareDialog.Compare', 'Compare');
+	private static readonly CancelButtonText: string = localize('schemaCompareDialog.Cancel', 'Cancel');
+	private static readonly SourceTextBoxLabel: string = localize('schemaCompareDialog.SourceLabel', 'Source File');
+	private static readonly TargetTextBoxLabel: string = localize('schemaCompareDialog.TargetLabel', 'Target File');
+	private static readonly DacpacRadioButtonLabel: string = localize('schemaCompare.dacpacRadioButtonLabel', 'Dacpac');
+	private static readonly DatabaseRadioButtonLabel: string = localize('schemaCompare.databaseButtonLabel', 'Database');
+	private static readonly SourceRadioButtonsLabel: string = localize('schemaCompare.sourceButtonsLabel', 'Source Type');
+	private static readonly TargetRadioButtonsLabel: string = localize('schemaCompare.targetButtonsLabel', 'Target Type');
+	private static readonly NoActiveConnectionsLabel: string = localize('schemaCompare.NoActiveConnectionsText', 'No active connections');
+	private static readonly SchemaCompareLabel: string = localize('schemaCompare.dialogTitle', 'Schema Compare');
+
+	public dialog: azdata.window.Dialog;
+	private schemaCompareTab: azdata.window.DialogTab;
+	private sourceDacpacComponent: azdata.FormComponent;
+	private sourceTextBox: azdata.InputBoxComponent;
+	private sourceFileButton: azdata.ButtonComponent;
+	private sourceServerComponent: azdata.FormComponent;
+	private sourceServerDropdown: azdata.DropDownComponent;
+	private sourceDatabaseComponent: azdata.FormComponent;
+	private sourceDatabaseDropdown: azdata.DropDownComponent;
+	private sourceNoActiveConnectionsText: azdata.FormComponent;
+	private targetDacpacComponent: azdata.FormComponent;
+	private targetTextBox: azdata.InputBoxComponent;
+	private targetFileButton: azdata.ButtonComponent;
+	private targetServerComponent: azdata.FormComponent;
+	private targetServerDropdown: azdata.DropDownComponent;
+	private targetDatabaseComponent: azdata.FormComponent;
+	private targetDatabaseDropdown: azdata.DropDownComponent;
+	private targetNoActiveConnectionsText: azdata.FormComponent;
+	private formBuilder: azdata.FormBuilder;
+	private sourceIsDacpac: boolean;
+	private targetIsDacpac: boolean;
+	private database: string;
+	public dialogName: string;
+
+	constructor(public ownerUri?: string) {
```

Comment:
Constructor but you don't do anything with the result?

Reply:
It's removed now. This was leftover from earlier experimentation.